src/views/AcademicCheckIn/components/ConfirmCheckIn/index.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Typography, Grid } from '@material-ui/core';
+import { gordonColors } from 'theme';
+
+const ConfirmCheckIn = ({
+  emergencyContact1,
+  emergencyContact2,
+  emergencyContact3,
+  personalPhone,
+  demographic,
+}) => {
+  const cyan = gordonColors.primary.cyan;
+
+  const raceValues = [
+    demographic.nativeAmerican,
+    demographic.asian,
+    demographic.black,
+    demographic.hawaiian,
+    demographic.white,
+    demographic.none,
+  ];
+  const raceNames = ['Native American', 'Asian', 'Black', 'Hawaiian', 'White', 'Prefer not to say'];
+
+  const displayRace = () => {
+    var i;
+    var names = '';
+    for (i = 0; i < raceValues.length; i++) {
+      raceValues[i] === true ? (names += raceNames[i] + ' ') : (names += '');
+    }
+    return names;
+  };
```

Comment:
You could condense this quite a bit:
```suggestion
  const { ethinicity, ...raceValues } = demographic;

  const races = {
    nativeAmerican: 'Native American',
    asian: 'Asian',
    black: 'Black',
    hawaiian: 'Hawaiian',
    white: 'White',
    none: 'Prefer not to say',
  };

  const displayRace = () => races.filter((race) => raceValues[race]).join(' ');
```